### PR TITLE
176 demo account

### PR DESCRIPTION
**Pull request by alishaevn**
ref: https://github.com/scientist-softserv/webstore/issues/176

# expected behavior
- return the bottom margin to the TextBox component
- set up several more components to accept color setting variables
- expand the color palette to include 4 darker and 4 lighter shades of the primary, secondary, light and dark brand colors

### CODE REVIEW

File: src/components/TextBox/TextBox.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const TextBox = ({ addClass, alignment, size, style, text }) => {
       {paragraphs.map((par, index) => (
         <div className={`text-box--${alignment} ${addClass || ''}`} key={index}>
           <p
-            className={`text-box text-box--${size} mb-0`}
```
Comment (alishaevn) [inline src/components/TextBox/TextBox.jsx:15]: when we have multiple paragraphs, we want space between them. I initially had a conditional to remove the spacing from the last paragraph only, but felt that was too specific to our current use case. (part of the "about us" section).

in the future, this component could be used anywhere, and having the bottom margin is a good thing.

File: src/compounds/RequestList/RequestList.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import RequestItem, { requestPropTypes } from '../RequestItem/RequestItem'
 const RequestList = ({ requests }) => (
   <>
     <Title title='My Requests' size='medium' />
-    <div className='rounded overflow-hidden'>
+    <div className='rounded overflow-hidden mb-4'>
```
Comment (alishaevn) [inline src/compounds/RequestList/RequestList.jsx:10]: adds some spacing between the last item in the list, and the footer.

File: src/components/StatusBar/StatusBar.jsx [hunk 3]
```diff
@@ -1,37 +1,56 @@
...
-              ${index !== statusArray.length - 1 ? 'border-end' : ''}
-              ${statusLabel === apiRequestStatus ? 'active' : ''}`}
-            key={statusLabel}
-          >
-            <FontAwesomeIcon icon={statusIcon} />
-            {statusLabel}
-          </div>
-        )
-      })}
+const StatusBar = ({ backgroundColor, statusArray, apiRequestStatus, addClass }) => {
```
Comment (alishaevn) [inline src/components/StatusBar/StatusBar.jsx:5]: added `backgroundColor`

File: src/components/StatusBar/StatusBar.jsx [hunk 4]
```diff
@@ -1,37 +1,56 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import './status-bar.scss'
 
-const StatusBar = ({ statusArray, apiRequestStatus, addClass }) => (
-  <div className={`container ${addClass}`}>
```
Comment (alishaevn) [inline src/components/StatusBar/StatusBar.jsx:7]: the majority of this code is the same. I needed to create a new variable and function though, so the return body is no longer immediately being returned. a lot of this change is adding indention for that reason.

File: src/components/StatusBar/StatusBar.jsx [hunk 5]
```diff
@@ -1,37 +1,54 @@
...
+    <div className={`container ${addClass}`}>
+      <div className='row border'>
+        {statusArray.map((statusObject, index) => {
+          const { statusLabel, statusIcon } = statusObject
+          const border = index !== statusArray.length - 1 ? 'border-end' : ''
+
+          return (
+            <div
+              className={`status-bar-column col-3 py-2 gap-2 d-flex justify-content-center align-items-center
+              ${border} ${labelBgColor(index)}`}
```
Comment (alishaevn) [inline src/components/StatusBar/StatusBar.jsx:27]: deriving both of these classes from the new code above.

**Review by summer-cook — APPROVED** [review]

**Pull request merged**